### PR TITLE
Bug 1813012: cleanup legacy etcd resources

### PR DESCRIPTION
**Pull request by retroflexer**
Remove resources and branches we don't need because maintenance is hard. These are causing maintenance headaches as they aren't used, but end up broken and stale. We will update our operator actually handle removal in running clusters.

Continued from #3626

### CODE REVIEW

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -73,49 +68,38 @@ then
 	touch cvo-bootstrap.done
 fi
 
-# during initial operator rollout phase this logic allows us to deploy the operator via CVO
-# in an `Unmanaged` no-op state. after all of the pieces have merged and the operator is
-# deemed stable we can remove this logic and the operator will be `Managed` by default.
-if [ ! -z "$CLUSTER_ETCD_OPERATOR_MANAGED" ]
+# TODO: host-etcd endpoint rendered by cluster-etcd-operator
```
Comment (abhinavdahiya) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:71]: can we add a link to etcd JIRA or etcd-operator github issue to track the TODO..

**Comment by hexfusion** [comment]
/close
work moved back to https://github.com/openshift/machine-config-operator/pull/1933

**Pull request closed without merging**